www/lendingbot.js

```diff
@@ -370,6 +376,30 @@ function doSave() {
     });
     localStorage.setItem('timespanNames', JSON.stringify(timespanNames))
 
+    // Bitcoin Display Unit
+    localStorage.displayUnitText = $('input[name="btcDisplayUnit"]:checked').val();
```

Comment:
Cool, that's fine then, I just tried refactoring that validation via the online editor, apologies if I made a small mistake or something -- should work though.